Update index.md (#831)
publiccodenet/about: activities/office-management/index.md

type: Index
---

# Office and building management

Our office houses the beating heart of the organization. We're all responsible for keeping it working and enjoyable.

The office is located at:
Keizersgracht 617
1017 DS Amsterdam

> We have an anti-squat contract (Bruikleenovereenkomst) with: K&O Group who manages the property on behalf of the owner, Hansainvest GmbH. The contract ends on 31 March 2021 with the possibility of extending it for on a month-to-month basis, until the building has found a new renter.

Reviewer: ```suggestion
We have an anti-squat contract (bruikleenovereenkomst) with K&O Group, which manages the property on behalf of the owner, Hansainvest GmbH. The contract ends on 31 March 2021 with the possibility of extending it on a month-to-month basis, until the building has found a new renter.
```

* [Using our space guide](using-space.md): What roles spaces have and what their code of conduct is.

> * Signed contract, office handbook (information on keys, WiFi, trash schedule) and contact details of K&O Group [are on the Secret Drive.](https://drive.google.com/drive/folders/1PJfus4U_eGuHkFPrxW1TTHH8RRg4tGpi)

Reviewer: ```suggestion
* Signed contract, office handbook (information on keys, WiFi, trash schedule) and contact details of K&O Group [are on the Secret Drive](https://drive.google.com/drive/folders/1PJfus4U_eGuHkFPrxW1TTHH8RRg4tGpi).
```
